scheduler/src/cook/mesos/api.clj

```diff
@@ -499,6 +533,49 @@
                           cheshire/generate-string)))
       ring.middleware.json/wrap-json-params))
 
+(def SwaggerApiDocs
```

Comment:
OK 👍 If there's not a library to do it, we can at least generate both from the same data ourselves.  I'll look into this.